recording/src/nextcloud/talk/recording/Participant.py

```diff
@@ -420,6 +473,8 @@ def __init__(self, browser, nextcloudUrl, width, height, env, parentLogger):
 
         if browser == 'firefox':
             self.seleniumHelper.startFirefox(width, height, env)
+        elif browser == 'chromium':
+            self.seleniumHelper.startChromium(width, height, env)
```

Comment:
Same about moving the code to match the [order of the calls in _Talkbuchet-cli.py_](https://github.com/nextcloud/spreed/blob/eaee928da9aee60e567752908f3136b80b0157f3/docs/Talkbuchet-cli.py#L524-L527)